### PR TITLE
2.x Fix authentication tests

### PR DESCRIPTION
**Pull request by e0ipso**
This will bring back the authentication tests.

### CODE REVIEW

**Comment by e0ipso** [comment]
@amitaibu tests are green.

File: src/Plugin/resource/Field/ResourceFieldEntity.php [hunk 1]
```diff
@@ -280,7 +280,7 @@ public function access($op, DataInterpreterInterface $interpreter) {
       return FALSE;
     }
 
-    $access = $property_wrapper->access($op, $account);
+    $access = $interpreter->getWrapper()->value() !== FALSE && $property_wrapper->access($op, $account);
```
Comment (amitaibu) [inline src/Plugin/resource/Field/ResourceFieldEntity.php:288]: `$interpreter->getWrapper()->value() !== FALSE` - Why is this needed?

**Comment by amitaibu** [comment]
added a small question - rest looks good.

File: src/Plugin/resource/Field/ResourceFieldEntity.php (hunk 1 above)
Comment (e0ipso) [inline src/Plugin/resource/Field/ResourceFieldEntity.php:288]: If `$interpreter->getWrapper()->value() === FALSE` it means that the entity could not be loaded, thus checking properties on it will result in errors.

Ex: this happens when the embedded author is the anonymous user. Doing `user_load(0)` returns `FALSE`.

Comment (amitaibu) [inline src/Plugin/resource/Field/ResourceFieldEntity.php:288]: Ok, I think it's worth adding this as a comment.

Comment (e0ipso) [inline src/Plugin/resource/Field/ResourceFieldEntity.php:288]: Fixed in 9bdcd44.

**Comment by e0ipso** [comment]
@amitaibu please merge.
